xbmc/settings/lib/Setting.h

```diff
@@ -402,7 +402,7 @@ class CSettingString : public CTraitedSetting<std::string, SettingType::String>
 
   virtual bool AllowEmpty() const { return m_allowEmpty; }
   void SetAllowEmpty(bool allowEmpty) { m_allowEmpty = allowEmpty; }
-  bool AllowNewOption() const { return m_allowNewOption; }
+  virtual bool AllowNewOption() const { return m_allowNewOption; }
```

Comment:
Why this change?

Reply:
As originally intended (like AllowEmpty) but somehow got lost. I spotted it when checking the rest so slipped in a one word fix rather than yet another PR

Reply:
Okay.